app/stylesheets/dashboard/charts-container.scss

```diff
@@ -1,5 +1,16 @@
 @import '../config/colours';
+@import '../config/dimensions';
+@import '../config/type';
 
 .charts-container {
   background: $grey-light;
 }
+
+.tab-label {
```

Comment:
Right now it's used in one place in charts-container.jsx for the charts toggle label